onnxscript/function_libs/torch_lib/ops/nn.py

```diff
@@ -2645,6 +2645,18 @@ def aten_upsample_nearest3d(
     return _aten_upsample_output_size(self, size, "nearest", "asymmetric")
 
 
+@torch_op("aten::upsample_nearest3d.vec", trace_only=True)
+def aten_upsample_nearest3d_vec(
+    input: TReal,
+    output_size: INT64,
+    scale_factors: Optional[Sequence[float]] = None,
+) -> TReal:
+    """upsample_nearest3d.vec(Tensor input, SymInt[]? output_size, float[]? scale_factors) -> Tensor"""
+
+    # TODO: Use scale_factors?
+    return _aten_upsample_output_size(input, output_size, "nearest", "asymmetric")
```

Comment:
For AHNet, we get a failure because output_size is missing here: is it supposed to be an optional input? The calling node in exported model looks as below:
```
%"upsample_nearest3d_7"<FLOAT,[1,2,128,128,128]> ⬅️ pkg.onnxscript.torch_lib::_aten_upsample_output_size(%"conv3d_121") {mode=nearest, 
coordinate_transformation_mode=asymmetric}
```

Reply:
Fixed